ClosedXML/Excel/XLClearOptions.cs

```diff
@@ -11,10 +11,11 @@ public enum XLClearOptions
         ConditionalFormats      = 1 << 3,
         Comments                = 1 << 4,
         DataValidation          = 1 << 5,
+        MergedRanges            = 1 << 6,
 
         AllFormats = NormalFormats | ConditionalFormats,
         AllContents = Contents | DataType | Comments,
-        All = Contents | DataType | NormalFormats | ConditionalFormats | Comments | DataValidation
+        All = Contents | DataType | NormalFormats | ConditionalFormats | Comments | DataValidation | MergedRanges
```

Comment:
I've just noticed that in order to match with `XLCellUsedOptions` `MergedRanges` flag must be added to `AllContents` as well.

Reply:
Oh ok, that answers my other question.